### DISCUSSION TITLE
My Student Developer Pack application was rejected

### DISCUSSION THREAD

**Discussion by newt239** [community/community#51322] [GitHub Education] [answered]
Hey, I'm a Japanese High School Student. I tried Student Developer Pack application 17 times, but all times rejected.
I have received following message:

> You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
> Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
> My certificate is written in Japanese, so I put my name, the school name and the expiry date of the certificate in English next to the original.

I have only this dated student ID and couldn't provide any other proof.

I have tried everything I can try and have also checked the related GitHub Discussions threads. Furthermore, I have opened a support ticket but have not received a response for over a week.

Can I have my application reviewed by a person? I want human checks, not AI checks.

Best regards.

**Comment by samus-aran** [comment]
Hi @newt239 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process. After you have updated your profile name please log out of GitHub and log back before re-applying 👍 

Let us know how you get on.

**Reply by philip01010101** [reply]
However, I suggest that you review the criteria for the Student Developer Pack carefully to make sure you meet all the requirements. If you believe that you meet the requirements and your application was wrongly rejected, you can reach out to the support team for the Student Developer Pack to inquire about the reason for the rejection and ask for further guidance on how to proceed.

**Reply by newt239** [reply]
Thanks for replies.
Of course I have tried changing my username to my real name, but the result did not change. Also, I have always applied using the email address given to me by my school. ( ed.jp )
My school is a general high school so I should meet the requirements. Can you tell me how I can contact the support team to find out the reason for the rejection?

**Reply by newt239** [reply]
@samus-aran @philip01010101 Sorry, I need your help. Please teach me what should I do for next.

**Comment by newt239** [comment, marked as answer]
I was passed this review yesterday. Official documents and guides are very unclear, so I will share what I was tried.

- change username on my profile to my real name.
- change location on my profile to my school located prefecture instead of my house.
- after update my profile, sign out from GitHub.
- after for a while, sign in again and resend application.

**Reply by chrisa530** [reply]
Is there a number I can reach Github. I made a ticket 16 days ago and no one has yet to answer me.

**Reply by samus-aran** [reply]
Thank you for sharing your feedback @newt239. So glad you were able to successful get verified. 🎉 In the name of improving our documentation what specifically would like the documentation to provide when you receive an unsuccessful application?

**Reply by newt239** [reply]
I think that ambiguous words used in the possible rejected reason confuse us. I want you to write the rejected reason clearly. And should write the advise that  resend application after sign out and in again.

**Reply by samus-aran** [reply]
Absolutely that makes sense! Thank you for this. 🙇

**Comment by fazalhadiqua** [comment]
Dear Sir /Madam 
I applied for Git hub student pack but got rejected 2 times . 
I am currently enrolled in Phd(CS )program . Also attached my documents as well . 
Kindly help in this regard

**Reply by fazalhadiqua** [reply]
please reply

**Reply by newt239** [reply]
Sorry, I'm a not GitHub staff, so I can't help you.
This post was alrady solved, so official staff may not see this page.
I recommend you to post a new thread in another place.

**Comment by jawad-arshad** [comment]
I was rejected just one time now he says me like :
Hi, Jawad Arshad! You were last verified as a student on Aug 31, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

and now try to submit again the continue button is disabled.

**Reply by o0kevinbt730o** [reply]
Hi, I have the same issue, do you have any update or ways on how to fix it?

**Reply by vickylam288** [reply]
I have the same issue too

**Comment by arun-pratap-singh** [comment]
![Screenshot 2024-02-26 194623](https://github.com/community/community/assets/147536994/af9e6743-de19-4d20-bf1b-e9b7be4c3fde)

Hey @newt239, I'm a Chandigarh University of Panjab Student. I tried the Student Developer Pack application 17 times, but all times rejected.
I have received the following message :

"Common verification problems and suggested solutions are documented here: [Why wasn't my application approved?]

(https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)."
